app/javascript/App/containers/newsfeed/actions.js

```diff
@@ -4,7 +4,35 @@ import { createEntityActions, createFilterActions } from '../../lib/redux'
 const entityActions = createEntityActions(namespace)
 const filterActions = createFilterActions(namespace)
 
+const scrollBottomToReducer = (scrollBottom) => {
+  if (!scrollBottom) return { type: null }
+
+  return { type: 'FETCH_MORE_NEWS_FEED' }
+}
+
+const onScrollNewsFeedMobile = (e) => {
```

Comment:
I think it would be nice to have a single action creator for desktop & mobile, which does the mobile detection (rather than doing that from the component)